src/api/controllers/auth.js

```diff
@@ -132,6 +137,61 @@ class AuthController {
     return res.status(200).send({ status: 200 });
   }
 
+  /**
+   * signup controller
+   * @param {Object} req  - Response
+   * @param {Object} res  - Response
+   * @param {Function} next -Next
+   * @returns {Object} The response object
+   */
+  static async login(req, res) {
+    joi
+      .validate(req.body, Validation.loginShema)
```

Comment:
Could you do this in middlewares?

Reply:
There is a middleware for body validation already. You can use it. Put your schema in `helpers/validationSchemas.js` and use it from `middleware/validateBody.js`